database/postgres/postgresData.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs');
+const Reservation = require('./Reservation.js');
+const Mapper = require('./Mapper.js');
+const Restaurant = require('./Restaurant.js');
+const faker = require('faker');
+const categories = require('../categories.json');
+
+/*
+10,000,000 restaurant id's --> with a random number of
+tables between 10 to 30 and seats between 10 to 30 times 5 to 10 with a random number of guests between 1-6.
+--> with a longitude and latitude
+--> with at least 1 reservation today
+--> created at
+--> updated at
+
+sample object: {
+  "_id":"5dfaed3c3144ba52b6c2b2b6", (don't need to produce)
+  "restaurant_id":99,
+  "customer_name":"Reina Botsford III",
+  "reservation_time":"2019-12-20T06:43:40.747Z",
+  "guests":3,
+  "created_at":"2019-12-19T03:23:40.758Z",
+  "updated_at":"2019-12-19T03:23:40.758Z",
+  "__v":0 (don't need to produce)
+}
+*/
+const dataGenerator = (path) => {
+
+  const ws = fs.createWriteStream(path);
+  ws.write(`restaurant_id,customer_name,reservation_time,guests,created_at,updated_at\n`);
+
+const RESERVATIONS_TO_CREATE = 10000000;
+
+//  RESERVATIONS
+//  we're going to use random number for restaurant_id so we don't end up with same
+//    number of reservations in each restaurant
+var res = [];
+
+for (var i = 0; i < RESERVATIONS_TO_CREATE; i++) {
+  res[i] = {};
+  res[i].restaurant_id = Math.floor(Math.random() * 1000000) + 1;
+  res[i].customer_name = faker.name.findName();
+  //  create random time between now and 3 days from now (4320 minutes)
+  var minutes = getRandomInteger(1, 4320);
+  var reserveTime = new Date();
+  reserveTime.setMinutes( reserveTime.getMinutes() + minutes );
+  console.log(minutes, reserveTime);
+  res[i].reservation_time = reserveTime;
+  console.log(res[i].reservation_time);
+  res[i].guests = Math.floor(Math.random() * 5) +1;
+}
+
+const entry = `${restaurant_id};${customer_name};${reservation_time};{${guests}};${created_at};${updated_at}\n`;
+
+}
+
+//  clear any existing reservations
+Reservation.deleteMany({}, ()=>{});
+//  plug in array
+Reservation.create(res, ()=>{});
```

Comment:
if you are creating a csv file, how come you are trying to insert the res data here?